python/paddle/fluid/tests/unittests/CMakeLists.txt

```diff
@@ -260,11 +260,16 @@ if(WITH_DISTRIBUTE)
         py_test_modules(test_dgc_op MODULES test_dgc_op)
         py_test_modules(test_dgc_momentum_op MODULES test_dgc_momentum_op)
         py_test_modules(test_dgc_optimizer MODULES test_dgc_optimizer)
+    else()
```

Comment:
?else?

Reply:
if(WITH_DGC)
else() # represent not WITH_DGC, remove dist dgc test